scss/config/_global-settings.scss

```diff
@@ -12,9 +12,102 @@ $decanter-default-settings: (
   breakpoint-helper-selector: #breakpoint-helper,
 );
 
-/// @name Bourbon Settings
-///
-/// @group settings
-$bourbon: (
```

Comment:
My guess is that removing this override is what is causing the font and margin size scale to have gone wild. Please restore it unless you intentionally went back to the 1.25 ratio instead of the 1.33 ratio.